node/src/command.rs

```diff
@@ -59,9 +60,58 @@ impl SubstrateCli for Cli {
 	}
 }
 
+#[cfg(test)]
+pub mod set_db_tests {
+	use crate::cli::Cli;
+	use crate::command::maybe_set_db;
+	use sc_cli::{Database, SubstrateCli};
+	use std::borrow::BorrowMut;
+
+	#[test]
+	fn maybe_set_db_changes_config_when_no_db_is_set() {
+		let mut cli = Cli::from_args();
+
+		// Start with no value set
+		assert_eq!(cli.run.import_params.database_params.database, None);
+		maybe_set_db(cli.borrow_mut());
+
+		// Expect it is now set to ParityDB
+		assert_eq!(cli.run.import_params.database_params.database, Some(Database::ParityDb));
+	}
+
+	#[test]
+	fn maybe_set_db_does_nothing_when_db_already_set() {
+		let mut cli = Cli::from_args();
+
+		// Set the value to something besides ParityDB
+		cli.run.import_params.database_params.database = Some(Database::RocksDb);
+		assert_eq!(cli.run.import_params.database_params.database, Some(Database::RocksDb));
+
+		maybe_set_db(cli.borrow_mut());
+
+		// Expect that the value is unchanged
+		assert_eq!(cli.run.import_params.database_params.database, Some(Database::RocksDb));
+	}
+}
+
+///Use ParityDB unless the user specified otherwise
+fn maybe_set_db(cli: &mut Cli) {
+	//Set the DB to ParityDB if needed
+	match &mut cli.run.import_params.database_params.database {
+		Some(_) => {}, // The user specified a database, so do nothing
+		db => {
+			// No Database set, default to ParityDB
+			*db = Some(Database::ParityDb);
+		},
+	};
+}
+
 /// Parse and run command line arguments
 pub fn run() -> sc_cli::Result<()> {
-	let cli = Cli::from_args();
+	let mut cli = Cli::from_args();
+
+	//Set the DB to ParityDB if needed
+	maybe_set_db(cli.borrow_mut());
```

Comment:
```suggestion
	maybe_set_db(&mut cli);
```
Calling `borrow_mut` through the `BorrowMut` trait is equivalent, but not super common/idiomatic. (The `Borrow`/`BorrowMut` traits are generally used only in generic contexts).